baselines/asset/src/v1/asset_service_client.ts.baseline

```diff
@@ -92,14 +157,16 @@ export class AssetServiceClient {
   constructor(opts?: ClientOptions) {
     // Ensure that options include all the required fields.
     const staticMembers = this.constructor as typeof AssetServiceClient;
-    const servicePath = opts?.servicePath || opts?.apiEndpoint || staticMembers.servicePath;
+    // See: https://google.aip.dev/auth/4114
+    const [cert, key] = detectClientCertificate(opts);
+    const servicePath = opts?.servicePath || opts?.apiEndpoint || this.mtlsServicePath(cert, key);
```

Comment:
It looks like the logic should be:
```
const servicePath = opts?.apiEndpoint || this.mtlsServicePath(cert, key);
```
User provided endpoint always comes first.

Reply:
This is fixed in most recently pushed code:

` `if (servicePath !== staticMembers.{{ id.get("servicePath") }}) return servicePath;`